source/fe/fe_nedelec.cc

```diff
@@ -207,9 +206,30 @@ FE_Nedelec<dim>::FE_Nedelec(const unsigned int order)
       default:
         Assert(false, ExcNotImplemented());
     }
+
+  // We need to initialize the dof permuation table and the one for the sign
+  // change.
+  initialize_quad_dof_index_permutation_and_sign_change();
+
+  /*
+   * Throw an exception on cells with non-standard faces
+   */
+  this->throw_exception_on_occurrance_of_nonstandard_faces = true;
```

Comment:
For this and other elements where the implementation is missing, this option changes the default behaviour (which presently is basically "do nothing"), does not it? 

This may not be a bad thing to do since results with this element and a given non-standard mesh will likely be incorrect, but this will at least break some tests...

Reply:
Yep, I am currently going over the broken tests and need to find out why exactly they failed.